pset-functions.js

```diff
@@ -115,6 +230,37 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+// Parent function that reverses the input word and then compares the output to the originally input word to see if they match
+function isPalindrome(word){
+
+// Nested function that reverses the input word
+  function reverseString(word) {
```

Comment:
Absolutely nothing wrong with creating a function with a function block; it is exactly how methods are made after-all. Great use of string methods and if/else block to return different values! [Here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L163) is a solution that is similar for your benefit.